eventdata/runners/kibana_runner.py

```diff
@@ -56,7 +56,7 @@ def kibana(es, params):
     sum_hits = 0
     max_took = 0
     for r in result["responses"]:
-        hits = r["hits"]["total"]
+        hits = r.get("hits", {}).get("total", 0)
```

Comment:
Perhaps the handling for hits-as-a-dict should be moved into the isinstance(hits, dict) block below?

Reply:
`hits` is always a dict, it's that `total` can either be a dict or an int?

Reply:
Since hits is always a dict, maybe the conditional below (line 60) isn't needed?

Reply:
Ok, to be more explicit: `r["hits"]` is always a `dict`but not the variable `hits` (maybe we should provide a different name...)

Reply:
Ah! thanks for the explanation.